src/devices/ovrheadset/OVRHeadset.cpp

```diff
@@ -4,7 +4,10 @@
  * CopyPolicy: Released under the terms of the LGPLv2.1 or later, see LGPL.TXT
  */
 
-
+#define AXIS_COUNT 8
+#define STICK_COUNT 2
+#define BUTTON_COUNT 13
```

Comment:
Please do not use `#define`, you can use  `static constexpr size_t AXIS_COUNT = 8;` (or `int`, or whatever it is expected) that works pretty much in the same way, but it is type safe.

Also please move these below the `#include`s